lib/smart_answer_flows/coronavirus-employee-risk-assessment.rb

```diff
@@ -85,7 +85,13 @@ def define
 
         next_node do |response|
           if response == "yes"
-            question :are_you_shielding?
+            if calculator.where_do_you_work == "retail"
+              outcome :workplace_should_be_closed
+            else
+              question :are_you_shielding?
+            end
+          elsif calculator.where_do_you_work == "retail"
```

Comment:
This might be better
```ruby
          work_in_retail = calculator.where_do_you_work == "retail"

          if response == "yes" && work_in_retail
             outcome :workplace_should_be_closed
          elsif response == "yes"
             question :are_you_shielding?
          elsif work_in_retail
             outcome :go_back_to_work
          else
            outcome :workplace_should_be_closed
          end
```

Reply:
Updated 👌🏼